cli/command/command.go

```diff
@@ -221,6 +229,8 @@ func Populate(context *project.Context, c *cli.Context) {
 		context.Log = true
 	} else if c.Command.Name == "up" {
 		context.Log = !c.Bool("d")
+		context.NoRecreate = c.Bool("no-recreate")
+		context.ForceRecreate = c.Bool("force-recreate")
```

Comment:
While your comment is most likely valid, I think it's out of the scope of this PR.  Basically all options from the CLI are currently set in the Context and not passed as options.  We can make that change, but I don't think it's needed for this PR.